streams/shared/src/main/scala/zio/stream/ZStream.scala

```diff
@@ -2923,6 +2930,49 @@ class ZStream[-R, +E, +A](val channel: ZChannel[R, Any, Any, Any, E, Chunk[A], A
       }
     }
 
+  /**
+   * Re-chunks the elements of the stream into at least chunks of `n` elements
+   * each. The last chunk might contain less than `n` elements unless specified
+   * otherwise.
+   */
+  def rechunkMin(n: Int, smallerLastChunk: Boolean = true)(implicit trace: ZTraceElement): ZStream[R, E, A] =
```

Comment:
`rechunkAtLeast`?

Reply:
Dropping elements if `smallerLastChunk` is `false`  seems pretty unexpected. I think the contract with chunking should be that it may change the chunk structure but shouldn't change the elements of the stream.